lib/ab/tests.rb

```diff
@@ -8,6 +8,14 @@ def before_picking_variant(&block)
       def after_picking_variant(&block)
         AssignedTest.after_picking_variant(&block)
       end
+
+      def logger=(logger)
+        @logger = logger
+      end
+
+      def logger
```

Comment:
Actually, maybe the logger should be on `Ab` instead of `Ab::Tests`. It looks weird when `Ab::MissingVariant` tries to look in `Ab::Tests` for a logger. We could also use the configuration block pattern for this gem:
```ruby
Ab.configure do |config|
  config.logger = MyLogger.new
end
```

:smile: